templates/blog/newsletter-form.html

```diff
@@ -0,0 +1,96 @@
+          <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" type="checkbox" id="insightscloudserver" name="insightscloudserver" value="Cloud and server" />
+          <label for="insightscloudserver">云和服务器</label>
+        </li>
+
+        <li class="p-list__item u-clearfix">
+          <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" type="checkbox" id="insightsdesktop" name="insightsdesktop" value="Desktop" />
+          <label for="insightsdesktop">桌面</label>
+        </li>
+
+        <li class="p-list__item u-clearfix">
+          <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" type="checkbox" id="insightsiot" name="insightsiot" value="Internet of Things" />
+          <label for="insightsiot">物联网</label>
+        </li>
+
+        <li class="p-list__item u-clearfix">
+          <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" id="insightsrobotics" name="insightsrobotics" value="Robotics" type="checkbox">
+          <label for="insightsrobotics">机器人</label>
+        </li>
+
+        <li class="p-list__item u-clearfix">
+          <input class="mktoLogicalField mktoCheckboxList mktoHasWidth" id="insightstutorials" name="insightstutorials" value="Tutorials" type="checkbox">
+          <label for="insightstutorials">教程</label>
+        </li>
+      </ul>
+
+      <div class="mktFormReq mktField">
+        <label>
+          <span class="u-no-margin--top">工作邮箱:<span>*</span></span>
+          <input required  id="Email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField mktoRequired" />
+        </label>
+      </div>
+
+      <div class="mktField mktLblRight">
+        <span class="mktInput mktLblRight">
+          <input class="u-no-margin--top mktFormCheckbox" name="canonicalUpdatesOptIn" id="CanonicalUpdatesOptIn" value="Receive updates" type="checkbox" />
+          <label for="CanonicalUpdatesOptIn" class="p-form__label u-no-margin--top">
+            <small>我同意接收有关Canonical的产品和服务信息。</small>
+          </label>
+          <span class="mktFormMsg"></span>
+        </span>
+      </div>
+
+      <p>
+        <small>在提交此表格的同时，我确认已阅读和同意的<a href="https://ubuntu.com/legal/data-privacy/newsletter">隐私声明</a>和<a href="https://ubuntu.com/legal/data-privacy">隐私政策。</a></small>
+      </p>
+
+      <div>
+        <span class="mktoButtonWrap p-card--content">
+          <button type="submit" class="u-no-margin--bottom">立即订阅</button>
+        </span>
+
+        <input type="hidden" name="Consent_to_Processing__c" value="yes" />
+
+        <input value="1959" class="mktoField mktoFieldDescriptor" name="lpId" type="hidden">
+
+        <input value="30" class="mktoField mktoFieldDescriptor" name="subId" type="hidden">
+
+        <input name="lpurl" value="https://pages.canonical.com/Insights-Subscription_Insights-Subscription-test.html?cr={creative}&amp;kw={keyword}" type="hidden">
+        <input value="3647" class="mktoField mktoFieldDescriptor" name="formid" type="hidden">
```

Comment:
you need 'formVid' as well
```suggestion
        <input value="3647" class="mktoField mktoFieldDescriptor" name="formid" type="hidden">
                <input value="3647" class="mktoField mktoFieldDescriptor" name="formVid" type="hidden">
```